spec/rails_helper.rb

```diff
@@ -12,6 +12,9 @@
 # Add additional requires below this line. Rails is not loaded until this point!
 
 Capybara.javascript_driver = :poltergeist
+Capybara.register_driver :poltergeist do |app|
+  Capybara::Poltergeist::Driver.new(app, timeout: 1.minute)
```

Comment:
There were three specific feature tests that were timing out due to the palette generation.I'll figure out how to change the timeout just for those tests.